### DISCUSSION TITLE
Cover art

### DISCUSSION THREAD

**Discussion by jipite** [rern/rAudio#1516] [Feature Requests]
Hello

I don't have the cover of audio cd or song in webradio. Why ? Is it a bug or is it not present ?

Thanks for your work

**Comment by R101G** [comment]
It relies on the station sending the metadata, which not all of them do. If they do, rAudio looks for a cover image to match the track title.

If it works for the default Radio Paradise streams, it's probably that there is no metadata being sent by the stations you listen to.

If you see the metadata but no cover image, something might be blocking access to last.fm etc.
